keyboards/ziptyze/lets_split_v3/info.json

```
@@ -0,0 +1,298 @@
+{
+    "manufacturer": "ziptyze",
+    "keyboard_name": "Let's Split v3",
+    "maintainer": "ziptyze",
+    "board": "GENERIC_RP_RP2040",
+    "bootloader": "rp2040",
+    "processor": "RP2040",
+    "features": {
+        "bootmagic": true,
+        "command": false,
+        "console": false,
+        "encoder": false,
+        "extrakey": true,
+        "mousekey": true,
+        "nkro": false,
```

not enabled by default
```suggestion
        "extrakey": true,
        "mousekey": true,
```